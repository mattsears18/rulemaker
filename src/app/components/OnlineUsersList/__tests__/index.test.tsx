import * as React from 'react';
import { render } from '@testing-library/react';

import { OnlineUsersList } from '..';
import { MockedProvider } from '@apollo/client/testing';

const mocks = [];

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<OnlineUsersList  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <MockedProvider mocks={mocks}>
        <OnlineUsersList />
      </MockedProvider>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
