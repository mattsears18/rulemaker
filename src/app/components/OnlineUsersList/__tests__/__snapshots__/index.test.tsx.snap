// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OnlineUsersList  /> should match snapshot 1`] = `
.c0 {
  border: 1px solid black;
  text-align: center;
}

.c0 ul {
  list-style: none;
}

<div
  class="c0"
>
  <h2>
    Online Users (
    )
  </h2>
  <p>
    Loading...
  </p>
</div>
`;
